lib/components/FantasySlider.jsx

```diff
@@ -193,7 +193,7 @@ const Wrapper = styled.div`
   }
 
   .swiper-pagination {
-    grid-template-columns: repeat(4, 1fr);
+    grid-template-columns: repeat(3, 1fr);
```

Comment:
@pedro199288 we would also need this change for mobile

Reply:
@begonaalvarezd thank you for the feedback! now is better